apps/web/src/views/PoolDetail/components/PoolStatus.tsx

```diff
@@ -0,0 +1,75 @@
+import { useTranslation } from '@pancakeswap/localization'
+import { Percent } from '@pancakeswap/swap-sdk-core'
+import { AutoColumn, AutoRow, Card, CardBody, Column, Text } from '@pancakeswap/uikit'
+import { useMemo } from 'react'
+import { PoolInfo } from 'state/farmsV4/state/type'
+import { useChainNameByQuery } from 'state/info/hooks'
+import { formatDollarAmount } from 'views/V3Info/utils/numbers'
+import { ChangePercent } from './ChangePercent'
+import { PoolTokens } from './PoolTokens'
+
+type PoolStatusProps = {
+  poolInfo?: PoolInfo | null
+}
+export const PoolStatus: React.FC<PoolStatusProps> = ({ poolInfo }) => {
+  const { t } = useTranslation()
+  const chainName = useChainNameByQuery()
+  const tvlChange = useMemo(() => {
+    if (!poolInfo) return null
+    return new Percent(
+      BigInt((Number(poolInfo.tvlUsd ?? 0) - Number(poolInfo.tvlUsd24h ?? 0)).toFixed(0)),
+      BigInt(Number(poolInfo.tvlUsd ?? 1).toFixed(0)),
+    )
+  }, [poolInfo])
+  const volChange = useMemo(() => {
+    if (!poolInfo) return null
+    return new Percent(
+      BigInt((Number(poolInfo.vol24hUsd ?? 0) - Number(poolInfo.vol48hUsd ?? 0)).toFixed(0)),
+      BigInt(Number(poolInfo.vol24hUsd ?? 1).toFixed(0)) || BigInt(1),
```

Comment:
I think the calculator have some mistake, percent = (val(after) - val(before))/val(before), so the denominator should be vol48hUsd?

Reply:
i found a existing util to do that, can use pancake-frontend/apps/web/src/utils/infoDataHelpers.ts:getPercentChange